site-manage/site-manage-util/util/src/java/org/sakaiproject/site/util/SiteParticipantHelper.java

```diff
@@ -344,7 +343,7 @@ public static void addParticipantsFromMemberships(Map participantsMap, AuthzGrou
 								participant.providerRole = member.getRole()!=null?member.getRole().getId():"";
 								participant.removeable = false;
 							}
-							participant.regId = "";
+							participant.regId = user.getDisplayId();
```

Comment:
The line below refers to member not user?

Reply:
Yeah, the `Member` object doesn't have the display ID field, but the `User` object (which is what was used to get the user Id to then retrieve the `Member` object) does. So we dump it out of the `User` object rather than the `Member` object.